sample/android-app/src/main/java/com/icerockdev/MainActivity.kt

```diff
@@ -32,6 +36,12 @@ class MainActivity : AppCompatActivity() {
         textView.text = text
         textView.typeface = Testing.getFont1().getTypeface(context = this)
 
+        val textColor = MR.colors.textColor.getColor(this).colorInt()
+            .run(Integer::toHexString)
+            .let { "#$it" }
+            .run(Color::parseColor)
```

Comment:
for what it? we got `Color` class. here for android exist `colorInt` https://github.com/icerockdev/moko-graphics/blob/master/graphics/src/androidMain/kotlin/dev/icerock/moko/graphics/ColorExt.kt#L10 that already give argb int value for setTextColor...for what here implemented this?

Reply:
It was for tests, was removed.